api-contract/contract.yaml

```diff
@@ -461,7 +625,25 @@ components:
           type: string
         imageProfile:
           type: string
-          format: binary     
+          format: binary   
+    Friend:
+      type: object
+      properties:
+        username:
+          type: string
```

Comment:
* needs to have an ID, something like friendId

It would be good to also have the createdAt, in case we want to know how long the friendship has lasted